src/Microsoft.Health.Dicom.Core/Models/Export/AzureBlobExportOptions.cs

```diff
@@ -0,0 +1,42 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.Health.Dicom.Core.Models.Export;
+
+internal sealed class AzureBlobExportOptions : IValidatableObject
```

Comment:
This has been moved to the `Core` lib as this type is needed by the serializer. This makes sense, as it would eventually be present in a client API for export.